advocacy_docs/partner_docs/VeritasGuide/05-UsingNetBackupForPostgreSQL.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Using NetBackup for PostgreSQL'
```

Comment:
Title and box on main page talking about using the integration.  All other guides use just the partner name. so would be
"Using Veritas"
Now that I see it I think it would be better to use product name.  Can you work with Jasmer to update the existing 4 guides to write it this way?